pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -473,6 +473,17 @@ func TestEnoughRequests(t *testing.T) {
 			name:                      "skip checking extended resource request with quantity zero via resource groups",
 			wantInsufficientResources: []InsufficientResource{},
 		},
+		{
+			pod: newResourcePod(
+				framework.Resource{
+					ScalarResources: map[v1.ResourceName]int64{
+						extendedResourceA: 1,
+					}}),
+			nodeInfo: framework.NewNodeInfo(newResourcePod(framework.Resource{
+				MilliCPU: 20, Memory: 30, ScalarResources: map[v1.ResourceName]int64{extendedResourceA: 1}})),
+			name:                      "skip checking resource request with quantity zero",
+			wantInsufficientResources: []InsufficientResource{},
```

Comment:
this test should pass without the changes you made, correct?

Reply:
@ahg-g No, the test won't pass because CPU/memory is over overcommit